nsv13/code/modules/jobs/security/aiming.dm

```diff
@@ -0,0 +1,162 @@
+/obj/item/gun/Initialize()
+	. = ..()
+	AddComponent(/datum/component/aiming)
+
+/mob/living/proc/aim_react()
+	var/list/options = list()
+	for(var/option in list("surrender", "ignore"))
+		options[option] = image(icon = 'nsv13/icons/effects/aiming.dmi', icon_state = option)
+	var/choice = show_radial_menu(src, src, options)
+	if(choice == "surrender")
+		emote("surrender")
+
+//Happens before the actual projectile creation
+/obj/item/gun/before_firing(atom/target,mob/user, aimed)
+	if(aimed)
+		if(chambered?.BB)
```

Comment:
Note that some guns (namely energy guns) use one magic projectile, I'd advise *against* doing this for those specific types.